packaging/packager_rpm.sh

```diff
@@ -64,4 +65,4 @@ else
 fi
 LIBPATH=".." COVERAGE="$COVERAGE" sh coverage_analysis.sh
 
-exit 0
+exit 0
```

Comment:
Also here

Reply:
I have restored all the empty spaces, the removal could have been the result of squashing multiple commits
After checking back on the code, I was my mistake. I can conclude that AutoPy is not compatible with Python version 3.9 or newer

Reply:
My initial comment here is just about the empty lines that were added, i.e. this diff should not exist

Reply:
Ah yes, I understand now, I have remove all the empty lines and restore them back to their original state